### PR TITLE
doc/cephadm: add steps to the adoption process

### PR DESCRIPTION
**Pull request by gurubert**
There are a few steps missing the get the full orchestrator experience.

Signed-off-by: Robert Sander <gurubert@gurubert.de>

### CODE REVIEW

File: doc/cephadm/adoption.rst [hunk 1]
```diff
@@ -154,6 +154,14 @@ Adoption process
...
       cephadm adopt --style legacy --name osd.2
 
+#. Set the default container image for the cluster by running an orchestrated update
+
+   .. prompt:: bash #
+
+      ceph orch upgrade start $IMAGE
+
+   This will set the config property container_image.
+
```
Comment (anthonyeleven) [inline doc/cephadm/adoption.rst:164]: Double backquotes around `container_image`

File: doc/cephadm/adoption.rst [hunk 2]
```diff
@@ -211,3 +219,22 @@ Adoption process
 
 #. Check the output of the command ``ceph health detail`` for cephadm warnings
    about stray cluster daemons or hosts that are not yet managed by cephadm.
+
+#. Remove ceph packages from the host except for ceph-common to keep ceph commandline tools.
```
Comment (anthonyeleven) [inline doc/cephadm/adoption.rst:223]: Ceph capital

**Review by anthonyeleven — CHANGES_REQUESTED** [review]

**Review by adk3798 — APPROVED** [review]

File: doc/cephadm/adoption.rst (hunk 1 above)
Comment (gurubert) [reply doc/cephadm/adoption.rst:164]: commit 10dd35e1018e8456ff71efabcab1ce66e5f214ed

File: doc/cephadm/adoption.rst (hunk 2 above)
Comment (gurubert) [reply doc/cephadm/adoption.rst]: commit dc20a0dea47448aeb8d66b0dd37e541298200377

**Comment by anthonyeleven** [comment]
LGTM, but PR and commit message should be like "doc/cephadm: Add steps to the adoption process"

**Review by anthonyeleven — APPROVED** [review]

**Comment by adk3798** [comment]
@gurubert FYI, there's a required check in our github CI that all commits have a signed-off-by line in them. Before merging we have to either add signatures to the unsigned commits or squash the commits (preferable in this case I think) then make sure the remaining commit is signed.

Thanks

**Comment by gurubert** [comment]
OK, I will create a new pull request that is properly signed off.

**Pull request closed without merging**